src/MaterialDesignThemes.Wpf/DialogHost.cs

```diff
@@ -405,6 +409,50 @@ private static void IsOpenPropertyChangedCallback(DependencyObject dependencyObj
         })));
     }
 
+    
+    private void ListenForWindowStateChanged(Window? window)
+    {
+        window ??= Window.GetWindow(this);
```

Comment:
Not sure we need the `??=` as the only caller used this same call to try and get the window, so if that is null, this will very likely return null a second time.
```suggestion
```